akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphBroadcastSpec.scala

```diff
@@ -214,6 +216,25 @@ class GraphBroadcastSpec extends AkkaSpec {
       upsub.expectCancellation()
     }
 
+    "alsoTo must broadcast" in assertAllStagesStopped {
+      val p, p2 = TestSink.probe[Int](system)
+      val (ps1, ps2) = Source(1 to 6).alsoToMat(p)(Keep.right).toMat(p2)(Keep.both).run()
```

Comment:
I assume you mean if "Keep.right" is "Keep.both" in the example? I assume it'll return: `val ((ps0, ps1), ps2)`